### DISCUSSION TITLE
Adding users to Budibase

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#6838] [Help] [answered]
Hi,

is the option "Import users" disabled for now?

**Comment by R2bEEaton** [comment, marked as answer]
Yes, for now. Improved User and Role features are on the 2022 roadmap.

https://github.com/Budibase/budibase/projects/10
#5509 is TBD and #5320 is scheduled for October.

**Reply by melohagan** [reply]
Thanks @R2bEEaton 

The **Import users** specifically will actually be available as part of the user group enhancement in the next release, coming up in the next couple of weeks :)

**Reply by R2bEEaton** [reply]
Even better, thanks to you and the team for all your hard work! 😃
